art-community-platform/backend/app/user/models.py

```diff
@@ -1,11 +1,16 @@
+import datetime
+
+import django.utils.timezone
 from django.db import models
 
 
 class User(models.Model):
+    name = models.CharField(max_length=100, default='Artie')
+    birthdate = models.DateField(default=django.utils.timezone.now)
```

Comment:
Because name and birthdate fields are mandatory, there is no need to have default values for these 2 fields.
However, Django is currently forcing us to add default values to these 2 fields since there are some previous records not having these 2 fields in the database.
As a result, for now there is no harm to have default values, but after old database records are deleted (in the future), it is better to remove these default values.

Reply:
Cleaning the database should be done in future, because probably new fields will be added to the User model in following days. As mentioned in the comment above, there is no harm to have default values for now.